src/api/Shrooms.Presentation.Api/Controllers/PostController.cs

```diff
@@ -92,6 +92,11 @@ public async Task<IHttpActionResult> CreatePost(CreateWallPostViewModel wallPost
                     notif.Notify(createdPost, userHubDto);
                 }, GetOrganizationName());
 
+                _asyncRunner.Run<NewMentionNotifier>(notif =>
+                {
+                    notif.NotifyNewMentionInPostTitle(createdPost.Id, createdPost.User.FullName, postModel.Mentions);
```

Comment:
Maybe it's worth checking if `postModel.Mentions` is not empty before executing `asyncRunner`?